README.md

```diff
@@ -15,6 +15,8 @@ hook_request 方法会组装一个新的 xhr 请求翻译后的字幕，通过 x
 缺点：遇到原字幕和翻译字幕不一一对应的时候很 egg pain，目前使用 filter 根据时间戳来判断，复杂度上升到了 O(n\*a\*b)，不过好在 a 基本是 1 或者 2，而 b 是一个句子中单词的数量，不会太大，所以都可以近似看作常数。
 
 # Usage
+**UPDATE**: 修复和改进的 fork 版[一键安装](https://github.com/AndySuen/Y2BDoubleSubs/raw/master/hook_request.user.js)
```

Comment:
Thanks for the contribute! but it's not safe to approve an out-of-control source code link in README, I'll remove it later. Thanks again for fix this script.